sdks/python/apache_beam/runners/common.py

```diff
@@ -1200,6 +1404,13 @@ def process(self, windowed_value):
       self._reraise_augmented(exn)
       return []
 
+  def process_batch(self, windowed_batch):
```

Comment:
Here (or elsewhere) it seems we should document that this will only be called with the kind of batch that this DoFn supports, right?

Reply:
Just to make it clear that the producer is responsible for handling any type conversions?